ci-operator/templates/cluster-launch-e2e.yaml

```diff
@@ -192,6 +193,12 @@ objects:
           -e openshift_test_repo=${RPM_REPO} -e 'oreg_url=${IMAGE_FORMAT}' \
           playbooks/${CLUSTER_TYPE}/openshift-cluster/launch.yml
 
+        if [[ -n "${MINOR_UPGRADE}" ]]; then
+          target_branch="$( git describe | sed 's/^openshift-ansible-\([0-9]*\.[0-9]*\)\.[0-9]*-.*/\1/' )"
```

Comment:
`version you are targeting` - should this be an RPM version or smth else?

The idea here is to run upgrade playbooks keeping the version of RPMs the same, so there is no need for additional params except the flag.

A major upgrade test would look entirely different - as it needs ansible upgraded and git branch switched

Reply:
The reason this check doesn't work is because the ansible image isn't guaranteed to have a git repo in it (in master it isn't).  The ansible image is already pre-built.

This template is launched from a job, and the jobs are going to be branch specific.  Because the jobs are branch specific, your job definition should hardcode what the minor versions are.  Where are you expecting to get your source RPMs for the latest code from if not RPM_REPO (which was what the cluster was installed with)?

Reply:
Oh, I see. I assumed the git repo would remain there. Updated it to pass `UPGRADE_VERSION` instead.